Jenkinsfile

```diff
@@ -356,6 +354,39 @@ pipeline {
               }
             }
           }
+          stage('Release') {
+            options {
+              skipDefaultCheckout()
+              timeout(time: 12, unit: 'HOURS')
+            }
+            when {
+              beforeInput true
```

Comment:
I do not remember if it will provision an agent, I think so, I'd add the `beforeAgent true` too

Reply:
It does use the top-level agent. But I think it's good to add it explicitly, in case we change it in the future to use a specific worker.